src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.SymbolToProjectId.cs

```diff
@@ -175,10 +176,13 @@ internal partial class SolutionCompilationState
         else if (symbol.IsKind(SymbolKind.TypeParameter, out ITypeParameterSymbol? typeParameter) &&
                  typeParameter.TypeParameterKind == TypeParameterKind.Cref)
         {
```

Comment:
code that was never used.  i just removed it as it didn't make sense in the first place.